generator/src/test/java/com/scottlogic/deg/generator/generation/fieldvaluesources/RealNumberFieldValueSourceTests.java

```diff
@@ -195,11 +196,11 @@ void shouldSupplyInterestingValuesWhenBoundariesAreExclusiveAndClose() {
     void shouldSupplyInterestingNonBlacklistedValues() {
         givenLowerBound(-10, true);
         givenUpperBound(10, true);
-        givenScale(1);
+        givenScale(0);
```

Comment:
Why was the number of decimal places changed?

Reply:
As discussed, this appears to be a bit of an annoying test to fix so we have disabled it with the intent to fix it immediatly afrter getting this pr in to unblock other tasks